app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -194,3 +194,17 @@ span.btn a{ text-decoration: none; color: #333333}
 body.modal-open {
   overflow: visible;
 }
+
```

Comment:
I've added the comment, however I think the Redmine issue shouldn't be created until this is merged, as it's 100% "unactionable" (what happens if this PR never gets merged)